### ISSUE TITLE
PSNR value

### ISSUE THREAD

**Issue by RJL-666666** [benhenryL/Deblurring-3D-Gaussian-Splatting#15]
Why is the PSNR value of the test set higher than that of the training set? Is this reasonable?
![image](https://github.com/user-attachments/assets/dbde1e55-d24c-4da9-9c17-496ecfd07739)

**Comment by benhenryL** [comment]
Hello, this is because our model produces clean image. 
Testing images are clean and rendering is also clean, thus high PSNR, but all training images are blurry, leading to low PSNR (clean rendering vs. blurry GT).
Hope this helps. Thanks!
